andela/ah-frontend-space
# #167166265 Landing page

#### What does this PR do?
- Creates the landing page via route `http://localhost:8080/` or `https://ah-frontend-space-stagin-pr-16.herokuapp.com/`

#### Description of Task to be completed?
- create different components on the home page
- test all the created components
- make sure all the components are reusable
- make sure page is responsive to screen sizes.
#### How should this be manually tested?
- one can clone this repository and run `npm install` to update their dependencies.
- run `npm start` and it should display
or alternatively one can use this route `https://ah-frontend-space-stagin-pr-16.herokuapp.com/`

#### What are the relevant pivotal tracker stories?
[#167166265](https://www.pivotaltracker.com/story/show/167166265)

#### Screenshots
<img width="1440" alt="Screenshot 2019-07-11 at 20 35 32" src="https://user-images.githubusercontent.com/43015966/61072197-75dcfd00-a41b-11e9-9e25-b8a6f4b000ef.png">
